blog/2021-03-01-OptimizingCovid19VaccinationScheduling.adoc

```diff
@@ -0,0 +1,145 @@
+= Optimizing COVID-19 vaccination appointment scheduling
+:page-interpolate: true
+//:imagesdir: 2021/03/01
+:awestruct-author: pbrown and emmurphy
+//not sure if can have two authors. If not, we'll make it Paul.
+:awestruct-layout: blogPostBase
+:awestruct-tags:
+
```

Comment:
Make sure there is a twitter/facebook picture by adding this
```suggestion
:awestruct-share_image_filename: vaccination-scheduling.png
```

Reply:
I think the file name will need to change, see below

Reply:
We'll also need to add this line like this one (but with the correct url) as it's being published on RH developers first:
```
:awestruct-canonical_url: "https://developers.redhat.com/blog/2018/02/19/voxxed-days-zurich-2018/"
```

Reply:
Looks like RH developers won't publish first, so removing this canonical url.